libraries/Microsoft.Bot.Builder.Ai.LUIS/LuisRecognizer.cs

```diff
@@ -28,9 +29,10 @@ public class LuisRecognizer : IRecognizer
         /// <param name="luisModel">The LUIS model to use to recognize text.</param>
         /// <param name="luisRecognizerOptions">The LUIS recognizer options to use.</param>
         /// <param name="options">The LUIS request options to use.</param>
-        public LuisRecognizer(ILuisModel luisModel, ILuisRecognizerOptions luisRecognizerOptions = null, ILuisOptions options = null)
+        /// <param name="customHttpClient">an optional alternate HttpClient.</param>
```

Comment:
I would probably drop the word "custom" from this parameter and just go with calling it `httpClient`.